aten/src/ATen/native/transformers/cuda/sdp_utils.cpp

```diff
@@ -252,6 +253,7 @@ bool check_mem_efficient_hardware_support(sdp_params const& params, bool debug)
       return false;
     }
   }
+  return true;
```

Comment:
Instead of adding `return true` here, we should revert the following change made in a previous PR: https://github.com/ROCm/pytorch/blame/648f1961a9db71afea7701c416b61ef97fcc3651/aten/src/ATen/native/transformers/cuda/sdp_utils.cpp#L274

Reply:
Actually I'm more leaning to make a separate function because nested ifdef is not very readable.

Reply:
Maybe make that refactor in upstream, and keep the changes here minimal?

Reply:
Fixed in the latest commit